src/topp/SpectraMerger.cpp

```diff
@@ -99,10 +99,16 @@ class TOPPSpectraMerger :
     registerOutputFile_("out", "<file>", "", "Output mzML file with merged spectra.");
     setValidFormats_("out", ListUtils::create<String>("mzML"));
 
-    registerStringOption_("merging_method", "<method>", "block_method", "Method of merging which should be used.", false);
-    setValidStrings_("merging_method", ListUtils::create<String>("precursor_method,block_method"));
+    registerStringOption_("merging_method", "<method>", "average_gaussian", "Method of merging which should be used.", false);
+    setValidStrings_("merging_method", ListUtils::create<String>("average_gaussian,average_tophat,precursor_method,block_method"));
 
     registerSubsection_("algorithm", "Algorithm section for merging spectra");
+    
+    registerStringOption_("algorithm:average_gaussian:spectrum_type", "<spectrum_type>", "automatic", "Spectrum type of the MS level to be averaged", false, true);
```

Comment:
No. Unfortunately not. - The spectrum_type parameter is correctly read. Only the warning remains.